src/db/block.ts

```diff
@@ -12,6 +12,18 @@ export class BlockDB {
     return this.connection.select('*').from('blocks');
   }
 
+  async getByIndepHash(indepHash: string) {
+    const block = (
+      await this.connection.queryBuilder()
+        .select('*')
+        .from('blocks')
+        .where('id', '=', indepHash)
```

Comment:
not sure if "id" field in blocks is the "index_hash" from the endpoint specs - https://docs.arweave.org/developers/server/http-api#get-block-by-id